gradle/js-dce/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+import com.moowork.gradle.node.npm.NpmTask
+import org.jetbrains.kotlin.gradle.tasks.Kotlin2JsCompile
+
+buildscript {
```

Comment:
Looks like you don't need `buildscript` repositories anymore, because you have no classpath dependencies

Reply:
Good point, fixed.